src/perform_step/extrapolation_perform_step.jl

```diff
@@ -250,8 +256,14 @@ end
 function perform_step!(integrator,cache::RichardsonEulerCache,repeat_step=false)
   @unpack t,dt,u,f,p = integrator
   @unpack k,T = cache
-  prediction!(integrator,cache,dt)
-  prediction!(integrator,cache,dt/2)
+  if integrator.alg.threading
+    Threads.@threads for i in [1,2]
+      prediction!(integrator,cache,dt/i)
+    end
+  else
+    prediction!(integrator,cache,dt)
+    prediction!(integrator,cache,dt/2)
+  end
   T[2,2] = (2*T[2,1] - T[1,1])/(2-1)
```

Comment:
why only extrapolate to the second order?

Reply:
This is classic Richardson extrapolation. AitkenNeville is based on this algorithm only which goes to higher order using recurrence formula. I think going to higher order will make it AitkenNeville.